src/meson.build

```diff
@@ -24,7 +24,8 @@ executable('Hyprland', src,
     dependency('gl', 'opengl'),
     dependency('threads'),
     dependency('pango'),
-    dependency('pangocairo')
+    dependency('pangocairo'),
+    dependency('libexecinfo', required: false),
```

Comment:
Redundant with the following:
https://github.com/hyprwm/Hyprland/blob/607c77428be95f1b4caf0cdfd299bed5ce9b21c8/meson.build#L53
https://github.com/hyprwm/Hyprland/blob/607c77428be95f1b4caf0cdfd299bed5ce9b21c8/src/meson.build#L19

Also, libexecinfo as shipped on BSDs and Alpine 3.16 doesn't support pkg-config i.e., `libexecinfo.pc` is missing thus `dependency()` won't work.

Reply:
So should it be removed entirely, including `backtrace_dep`?

Reply:
`backtrace_dep` is still required to conditionally add `-lexecinfo` on BSDs or nothing on Linux/glibc.